### ISSUE TITLE
Alt+O fails to copy if command contains backslash \

### ISSUE THREAD

**Issue by sawyer07** [LukeSmithxyz/st#165]
If I open st and run 
`echo hello\ world` or `echo hello\world` then Alt+O, I see the command as an option to copy, but after selecting it nothing is put on the clipboard. `xclip -out selection clipboard | xxd` yields `Error: target STRING not available`

**Comment by lpanebr** [comment]
I can confirm that.

`alt+o` works as expected for the first command below, but fails for the other:
```sh
echo $PATH
echo $PATH |tr ":" "\n"
```

**Comment by LukeSmithxyz** [comment]
Duplicate of https://github.com/LukeSmithxyz/st/issues/157

**Comment by sawyer07** [comment]
I suspect that your having closed this as a duplicate of #157 might have been done in error. On the surface at least they do not appear to be the same issue (a forward slash in PS1 vs a backslash in command), and the issue here has been replicated by at least one other person.

The copyout feature was the reason I got into this terminal but it fails often. I'm not sure I've narrowed it down exclusively to backslashes but it's a consistent example. Apologies as I don't know all the etiquette here. If writing this doesn't automatically re-open the issue I'll plan to open a new one in the near future unless advised otherwise. Thanks for your work!
